src/Common/src/Internal/Runtime/MetadataBlob.cs

```diff
@@ -22,7 +22,7 @@ internal enum ReflectionMapBlob
         DefaultConstructorMap                       = 14,
         UnboxingAndInstantiatingStubMap             = 15,
         InvokeInstantiations                        = 16, // unused
-        PointerTypeMap                              = 17, // unused
```

Comment:
Reusing should be fine. We don't have to be backwards compatible with old classlibs right now.